### PR TITLE
wr-291: Rework latest articles section to look like on the design

### PR DESCRIPTION
**Pull request by drMangust**
image
image
image

### CODE REVIEW

File: frontend/src/pages/profile/components/article-short-card/article-short-card.tsx [hunk 1]
```diff
@@ -0,0 +1,68 @@
+import { type FC } from 'react';
+
+import { Icon } from '~/libs/components/icon/icon.js';
+import { Link } from '~/libs/components/link/link.js';
```
Comment (andanf-e) [inline frontend/src/pages/profile/components/article-short-card/article-short-card.tsx]:
```suggestion
import { Icon, Link } from '~/libs/components/components.js';
```

File: frontend/src/pages/profile/components/article-short-card/article-short-card.tsx [hunk 2]
```diff
@@ -0,0 +1,68 @@
...
+import { AppRoute } from '~/libs/enums/app-route.enum.js';
+import { LinkHash } from '~/libs/enums/link-hash.enum.js';
```
Comment (andanf-e) [inline frontend/src/pages/profile/components/article-short-card/article-short-card.tsx]:
```suggestion
import { AppRoute, LinkHash } from '~/libs/enums/enums.js';
```

Comment (drMangust) [reply frontend/src/pages/profile/components/article-short-card/article-short-card.tsx]: [c8e612f](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/330/commits/c8e612f540fcca4a4c908c195120393cbede2b44)

File: frontend/src/pages/profile/components/article-short-card/article-short-card.tsx (hunk 1 above)
Comment (drMangust) [reply frontend/src/pages/profile/components/article-short-card/article-short-card.tsx]: [c8e612f](https://github.com/BinaryStudioAcademy/bsa-2023-writorium/pull/330/commits/c8e612f540fcca4a4c908c195120393cbede2b44)

**Review by andanf-e — APPROVED** [review]

**Pull request merged**